tools/dependency/release_dates.py

```diff
@@ -38,32 +39,72 @@ def VerifyAndPrintLatestRelease(dep, repo, metadata_version, release_date):
   try:
     latest_release = repo.get_latest_release()
     if latest_release.created_at > release_date and latest_release.tag_name != metadata_version:
-      print(f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>: '
-            f'{latest_release.tag_name}@<{latest_release.created_at}>')
-  except github.UnknownObjectException:
+      print(Fore.YELLOW +
```

Comment:
Curious, does CI render ANSI color?

Reply:
It does render! https://dev.azure.com/cncf/envoy/_build/results?buildId=60325&view=logs&j=1c04237c-5f52-5959-5f83-5522b3556331&t=db3f3889-c922-566b-fa2e-4bc4948bc467

Reply:
Yes!
![image](https://user-images.githubusercontent.com/246317/101600173-c6111380-39af-11eb-9867-2b51cea21217.png)